app/Domain/Consignment/Repository/ConsignmentRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Domain\Consignment\Repository;
+
+use App\Models\Consignment;
+use App\Models\ConsignmentProduct;
+use Illuminate\Http\JsonResponse;
+
+class ConsignmentRepository
```

Comment:
Usually, the repository returns the models not responses. It's a bit of a different layer.
Repository - DOMAIN, response -> infrastructure